### DISCUSSION TITLE
Not able to use resume.json

### DISCUSSION THREAD

**Discussion by LeoPtls** [alshedivat/al-folio#2610] [Q&A]
Hi, 
I made some change on resume.json and I was not able to see my changes. So I endup using cv.yml. 
I think I most likely missed a step. Can someone help me ?

**Comment by george-gca** [comment]
Just tested out on the template and changing the file [assets/json/resume.json](https://github.com/alshedivat/al-folio/blob/master/assets/json/resume.json) works just fine. Did you make any change to [_layouts/cv.liquid](https://github.com/alshedivat/al-folio/blob/master/_layouts/cv.liquid)?
